frontend/src/metabase/visualizations/components/settings/ChartSettingFieldPicker.styled.tsx

```diff
@@ -65,3 +67,15 @@ export const SettingsIcon = styled(Icon)<SettingsIconProps>`
     color: ${color("brand")};
   }
 `;
+
+export const FieldPickerColorPicker = styled(ChartSettingColorPicker)`
+  margin-bottom: 0;
+  margin-left: 0.25rem;
+  ${ColorPill.Root} {
+    padding: 1px;
+  }
+  ${ColorPill.Content} {
+    height: 0.875rem;
+    width: 0.875rem;
+  }
```

Comment:
I'm game for that, though I've generally been encouraged to use emotion in places like this, rather than define new props/variants. But it feels like it might fit better here since we do this in 2 different places

Reply:
@npfitz @WiNloSt using emotion for styling inner parts is a hack that we use only because we can't agree on the design system

Reply:
Gotcha. We chatted on slack and I'm updating this with a "small" and "large" size